src/widgets/breadcrumb/breadcrumb.js

```diff
@@ -146,9 +146,9 @@ export default function breadcrumb({
 
   const cssClasses = {
     root: cx(suit(), userCssClasses.root),
-    noRefinement: cx(
-      suit({ modifierName: 'noRefinement' }),
-      userCssClasses.noRefinement
+    noRefinementRoot: cx(
+      suit({ modifierName: 'noRefinementRoot' }),
```

Comment:
The name of the modifier should still be `noRefinement` rather than `noRefinementRoot`.

Reply:
Nice catch!